test/kafka-sink-stalled-errors/mzcompose.py

```diff
@@ -0,0 +1,43 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
we might be able to place this test inside another workflow in an mzcompose, but i'm not sure which, I think we should ask @philip-stoev !

Reply:
if we keep this new mzcompose, it will need to be added to `ci/test/pipeline.template.yml`!

Reply:
I made a separate mzcompose because the parameters to `Kafka` needed to be different. 
https://github.com/MaterializeInc/materialize/pull/18009/files#diff-537e43fa738a41fbfdbda0f38466c9fb07f818f8961c36182c2cd66d5da5b50cR26-R30

Is it possible to have in an existing mzcompose with different Kafka init params?

Reply:
Added it to the `ci/test/pipeline.template.yml` for the timebeing

Reply:
Adding a separate Buildkite step for a small test is generally discouraged, as there is a lot of overhead in running such a test -- In https://buildkite.com/materialize/tests/builds/51507#0186e269-fe16-4faf-9fba-718dea36385f the actual test was 17 seconds with 2 min worth of setup and teardown.

However, putting this exacrt in `source-sink-errors` may require a lot of fiddling, so I guess a separate Buildkite step would be OK in this case, if `source-sink-errors` can not be made to work.

Reply:
Makes sense. Let me see if I can make it work with `source-sink-errors`

Reply:
Done!